service/history/api/startworkflow/api.go

```diff
@@ -80,7 +80,7 @@ type Starter struct {
 	visibilityManager          manager.VisibilityManager
 	request                    *historyservice.StartWorkflowExecutionRequest
 	namespace                  *namespace.Namespace
-	createLeaseForWorkflow     api.CreateLeaseForWorkflow
+	createLeaseFunc            api.CreateLeaseFunc
```

Comment:
```suggestion
	createLeaseFn            api.CreateLeaseFunc
```
I will keep saying; `Func` for type, `Fn` for var/fields.